active/ZEP0000.md

```diff
@@ -264,130 +276,220 @@ original and new ZEPs respectively. Process ZEPs may also have a status of
-do it – the important thing is to make sure that everyone knows when a ZEP is
-on the verge of acceptance, and give them a final chance to respond. If
-there’s some special reason to extend this final comment period beyond
-`7 days`, then that’s fine, just say so in the post. You shouldn’t do less
-than `7 days`, because sometimes people are travelling or similar and need
-some time to respond.
+The Reading Phase, or Request for Comments (RFC) phase, initiates the ZEP
+proposal process, marking the introduction of the proposal to the Zarr
+community.
 
-There may be a case that a ZEP didn’t attract needed attention towards it, the
-engagement from the community is low, and `7 days` pass by. In this case,
-the author(s) of the ZEP must make necessary efforts to spread the word about
-the ZEP through [Gitter] or using the organisation [discussions] feature of
-GitHub or, if needed, creating an additional issue in the [community] or [ZEPs]
-repository. In addition, the author(s) should get in touch with the
-[Zarr Steering Council] to prevent the case that the ZEP was accepted due to
-less participation from the community.
+1. **Stakeholder Engagement**:
+    - During this phase, stakeholders actively participate by expressing
+      concerns and opinions related to the proposal.
+    - The author uses the submitted PR or establishes a separate GitHub issue to
+      facilitate discussions, addressing questions, doubts, and concerns.
 
-If all the above options are exhausted by the author(s), then it’s the
-responsibility of the [Zarr Steering Council] to take the final decision on the
-ZEP.
+2. **Duration Definition**:
+    - The author communicates the duration of the Reading Phase when submitting
+      the ZEP, typically ranging from a suggested 3-4 weeks with flexibility
+      for exceptional circumstances.
+
+3. **Feedback Collection**:
+    - Authors anticipate and welcome suggestions from readers to enhance the
+      proposal, encouraging general comments to refine definitions, add URLs,
+      or improve overall readability.
+
+4. **ZIC Acknowledgment**:
+    - ZIC members publicly acknowledge their engagement during the Reading
+      Phase, ensuring a comprehensive understanding of community involvement.
+
+<!-- Mention Zarr-Spec devs in line: 315-322, after zarr-developers/governance/#44 is finalised -->
+
+5. **VETO Mechanism**:
+    - A VETO option is available to ZIC members if the proposal introduces
+      potential breaking changes to existing software implementations, serving
+      as a precautionary measure.
+
+Authors may utilize the submitted pull request (PR) for gathering comments,
+feedback, acknowledgment, and facilitating engagement from ZIC members during
+the Reading Phase.
+
+### Implementation Phase
+
+The Implementation Phase encompasses the timeline during which interested
+adopters actively work on integrating the proposal into their open-source
```

Comment:
Why should we limit implementations to open-source ones? ARC-GIS supports Zarr, for example.

```suggestion
adopters actively work on integrating the proposal into their
```